ReportPortal.ATM/ReporterLibrary/Reporters/ExtentReporterService.cs

```diff
@@ -0,0 +1,48 @@
+﻿using AventStack.ExtentReports;
+using AventStack.ExtentReports.Reporter;
+using ReporterLibrary.Interfaces.Reporters;
+using System;
+using System.IO;
+using System.Reflection;
+
+namespace ReporterLibrary.Reporters
+{
+    public class ExtentReporterService : IReporterService
+    {
+        private readonly ExtentReports _extenReport;
+        private ExtentTest _extentTest;
+
+        public ExtentReporterService()
+        {
+            var path = Path.Combine(Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location), "results'\'");
+
+            Directory.CreateDirectory(path);
+
+            _extenReport = new ExtentReports();
+            var _extentHtmlReporter = new ExtentSparkReporter(path);
```

Comment:
Let's remove the underscore from this variable, local variables in .NET are just camelCased:

`var extentHtmlReporter = new ExtentSparkReporter(path);`

Reply:
Changed to receive path as parameter